meta/runtime.yml

```diff
@@ -22,6 +22,10 @@ plugin_routing:
     nios_next_network:
       redirect: infoblox.nios_modules.nios_next_network
   modules:
+    consul_acl:
```

Comment:
It's best to move the deprecation to a separate PR. It also needs a separate changelog fragment (`deprecated_features`).

Reply:
Ok removed it for now from this PR completely.